### PR TITLE
added chart for pdf-generator

### PR DESCRIPTION
**Pull request by acn-sbuad**
## Description
Added helm chart for pdf generator based on browserless chrome .

## Verification
- [x] **Your** code builds clean without any errors or warnings
- [x] Manual testing done (required)

### CODE REVIEW

File: charts/pdf-generator/templates/middleware.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{{- range $middleware := .Values.middlewares }}
```
Comment (tjololo) [inline charts/pdf-generator/templates/middleware.yaml:1]: Is this file necessary?

File: charts/pdf-generator/values.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+resources:
+  requests:
+    cpu: 30m
+    memory: 128Mi
+
+image:
+  # Set "repository" name of your image for manual Helm install and upgrade.
+  repository: browserless/chrome
+  tag: latest
```
Comment (tjololo) [inline charts/pdf-generator/values.yaml:24]: It's recomended to not depend on latest as we have no control if there are breaking changes in latest. Should we rather depend on a pinned version and upgrade the chart when new versions are available?

**Review by tjololo — CHANGES_REQUESTED** [review]
For role, rolebinding and serviceaccount we are hardcoding the name, but for the other we depend on the name of the chart/deployment through the helpers. I think we should be consistent in how we name the resources. Also on these resources namespace is set to pdf but not on the other this could lead to the resources being deployed in different namespaces.

File: charts/pdf-generator/values.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  pullPolicy: Always
```
Comment (tjololo) [inline charts/pdf-generator/values.yaml:25]: Should we change the default pullPolicy to IfNotPresent?

Comment (acn-sbuad) [inline charts/pdf-generator/values.yaml:25]:
```suggestion
  pullPolicy: IfNotPresent
```

File: charts/pdf-generator/values.yaml (hunk 2 above)
Comment (acn-sbuad) [inline charts/pdf-generator/values.yaml:24]:
```suggestion
  tag: 1-puppeteer-19.2.2
```

File: charts/pdf-generator/templates/deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,59 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "name" . }}
+  namespace: {{ .Values.namespace }}
```
Comment (tjololo) [inline charts/pdf-generator/templates/deployment.yaml:5]: Sorry my bad! I just remembered that it's possible to use a built in variable in helm to control namespace, we can then use: `helm install --namespace whatever` to control the namespace it should be installed

```suggestion
  namespace: {{ Release.Namespace }}
```

File: charts/pdf-generator/values.yaml [hunk 5]
```diff
@@ -0,0 +1,34 @@
+replicaCount: 2
+
+environment: development
+
+namespace: pdf
```
Comment (tjololo) [inline charts/pdf-generator/values.yaml:5]: Remove unused variable
```suggestion
```

**Review by tjololo — APPROVED** [review]

**Pull request merged**